src/OpenTelemetry.Exporter.Geneva/MsgPackExporter/MsgPackLogExporter.cs

```diff
@@ -138,6 +138,8 @@ internal bool IsUsingUnixDomainSocket
 
     internal int SerializeLogRecord(LogRecord logRecord)
     {
+        // `LogRecord.State` and `LogRecord.StateValues` were marked Obsolete in https://github.com/open-telemetry/opentelemetry-dotnet/pull/4334
+#pragma warning disable 0618
```

Comment:
@cijothomas @CodeBlanch Okay with this change?

Reply:
Yes. Doing this suppress-warning, will be yet another validation that the main sdk has done everything in back-compat way!